packages/oranger/oranger.2.0.1/opam

```diff
@@ -21,6 +21,7 @@ depends: [
   "minicli" {with-test & >= "5.0.0"}
   "cpm" {with-test}
   "conf-gnuplot" {with-test}
+  "conf-bash" {build}
```

Comment:
is this package really using bash? I can only see `sh` used here

Reply:
The file `packages/oranger/oranger.2.0.1/files/compile_ranger.sh` has a `#!/bin/bash` shebang. And in the repository the file `install_ranger.sh` also has the bash shebang.

Reply:
`compile_ranger.sh` is not used anywhere as far as i can see, and `install_ranger.sh` is called directly using `sh` which bypasser the shebang

Reply:
I removed this for this package and the other `oranger` packages.